From a208dc70c9a5abdd9d2682b4f3453708d4d6cdf3 Mon Sep 17 00:00:00 2001
From: Elvis Angelaccio <elvis.angelaccio@kde.org>
Date: Thu, 16 Nov 2017 12:46:17 +0100
Subject: Fix build on FreeBSD

Summary: I broke the build on FreeBSD, this patch should fix it.

Reviewers: adridg, dfaure

Subscribers: #frameworks

Tags: #frameworks

Differential Revision: https://phabricator.kde.org/D8841
---
 src/core/listjob.cpp   | 2 +-
 src/core/mkpathjob.cpp | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)


diff --git a/src/core/mkpathjob.cpp b/src/core/mkpathjob.cpp
index a177805..c77a9fe 100644
--- a/src/core/mkpathjob.cpp
+++ b/src/core/mkpathjob.cpp
@@ -45,7 +45,7 @@ public:
         for (; i < basePathComponents.count() && i < m_pathComponents.count(); ++i) {
             const QString pathComponent = m_pathComponents.at(i);
             if (pathComponent == basePathComponents.at(i)) {
-                if (m_url.path() == QLatin1Char('/')) {
+                if (m_url.path() == QLatin1String("/")) {
                     m_url.setPath(m_url.path() + pathComponent);
                 } else {
                     m_url.setPath(m_url.path() + '/' + pathComponent);
@@ -64,7 +64,7 @@ public:
             for (; i < m_pathComponents.count(); ++i) {
                 const QString localFile = m_url.toLocalFile();
                 QString testDir;
-                if (localFile == QLatin1Char('/')) {
+                if (localFile == QLatin1String("/")) {
                     testDir = localFile + m_pathComponents.at(i);
                 } else {
                     testDir = localFile + '/' + m_pathComponents.at(i);
-- 
cgit v0.11.2

